crates/wasi-http/src/http_impl.rs

```diff
@@ -54,7 +63,7 @@ fn port_for_scheme(scheme: &Option<Scheme>) -> &str {
     }
 }
 
-impl WasiHttp {
+impl WasiHttpCtx {
```

Comment:
Is there a reason for the rename?

Reply:
just to follow the pattern in the other modules, and also to show that there is a breaking change to the API